server/zanata-war/src/test/java/org/zanata/ZanataRestTest.java

```diff
@@ -36,12 +40,13 @@
 
     protected static final URI MOCK_BASE_URI = URI.create("http://mockhost");
 
-    private ClientRequestFactory clientRequestFactory;
+    private ResteasyClientBuilder resteasyClientBuilder;
     protected final Set<Class<? extends ExceptionMapper<? extends Throwable>>> exceptionMappers =
             newHashSet();
     protected final Set<Object> resources = newHashSet();
     protected final Set<Class<?>> providers = newHashSet();
     protected final Set<Object> providerInstances = newHashSet();
+    protected Dispatcher dispatcher;
```

Comment:
It's just test code, so I wouldn't be too bothered if you left off the `private` modifier and made it package-protected, but why protected?

In fact, why convert it to a field?